src/app/tokens/TokenRow/index.tsx

```diff
@@ -39,7 +69,12 @@ export const TokenRow: FC<{
       </Td>
       <Td isNumeric width={'130px'} padding={'10px 16px 10px 20px'}>
         <Text whiteSpace={'nowrap'} textOverflow={'ellipsis'} overflow={'hidden'} fontSize={'15px'}>
-          {numberToString(ftToken.total_supply ? Number(ftToken.total_supply) : 0)}
+          {ftToken.total_supply
+            ? abbreviateNumber(
+                getFtDecimalAdjustedBalance(ftToken.total_supply, ftToken.decimals || 0),
+                2
+              )
+            : 'N/A'}
```

Comment:
I believe N/A is better than 0 as I don't think you can even have a token with a supply of 0